collection-scripts/gather_ceph_logs

```diff
@@ -27,6 +27,17 @@ for ns in $namespaces; do
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/rook/"${ns}".nfs.csi.ceph.com/log "${CSI_LOG_OUTPUT_DIR}"
     }
 
+    # collect logs from the csi pods if csi is installed using csi-operator
+    csi_operator_log_collection() {
+        dbglog "collecting csi logs from node ${node}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/cephcsi/"${ns}".rbd.csi.ceph.com-ctrlplugin "${CSI_LOG_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/cephcsi/"${ns}".rbd.csi.ceph.com-nodeplugin "${CSI_LOG_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/cephcsi/"${ns}".cephfs.csi.ceph.com-ctrlplugin "${CSI_LOG_OUTPUT_DIR}"
```

Comment:
@yati1998 if the required host path is not present `var/lib/cephcsi/"${ns}".cephfs.csi.ceph.com-ctrlplugin` ,  the other must gather collection wont fail, right?

Reply:
no it wont. Did you test the changes?

Reply:
Can we add a check below before calling the function, to make sure we collect it incase of csi-operator only?

Reply:
sure, we can discuss this offline